agent/src/agent/pipeline/config/stages/expression_evaluator.py

```diff
@@ -43,11 +43,14 @@ def _get_config(self) -> dict:
         expressions = []
         for key, val in self.pipeline.static_dimensions.items():
             expressions.append(get_value(self._get_dimension_field_path(key), f'"{val}"'))
-        timestamp_to_unix = get_convert_timestamp_to_unix_expression(self.pipeline.timestamp_type,
-                                                                     "record:value('/timestamp')",
-                                                                     self.pipeline.timestamp_format,
-                                                                     self.pipeline.timezone)
-        expressions.append(get_value('/timestamp', timestamp_to_unix))
+        if 'timestamp' in self.pipeline.config:
```

Comment:
for every influx pipeline we can set timestamp type as unix_ms and field name `time`